codec-http2/src/main/java/io/netty/handler/codec/http2/Http2FrameLogger.java

```diff
@@ -98,14 +98,14 @@ public void logSettings(Direction direction, ChannelHandlerContext ctx, Http2Set
         logger.log(level, "{} {} SETTINGS: ack=false settings={}", ctx.channel(), direction.name(), settings);
     }
 
-    public void logPing(Direction direction, ChannelHandlerContext ctx, ByteBuf data) {
+    public void logPing(Direction direction, ChannelHandlerContext ctx, long data) {
         logger.log(level, "{} {} PING: ack=false length={} bytes={}", ctx.channel(),
-                direction.name(), data.readableBytes(), toString(data));
+                direction.name(), 8, data);
```

Comment:
I don't think its necessary to log 8 bytes, because it always has to be 8 bytes.